libs/core/src/components/sage-link/readme.md

```diff
@@ -4,6 +4,7 @@ Link is mainly used as navigational element and usually appear within or directl
 
```

Comment:
These file updates can be removed

Reply:
I'll revert this back, not sure why it changed.